src/7-to-8/summary.rst

```diff
@@ -1,4 +1,4 @@
-.. _overview:
```

Comment:
unrelated: vague label which doesn't seem to be used anywhere at present, this will most likely conflict (e.g. intersphinx) so swapping it out for something more specific.